docker-compose.yml

```diff
@@ -11,18 +11,7 @@ services:
     depends_on:
       - postgres
       - redis
-    environment:
```

Comment:
Are you sure this works?

I thought you would have to do something like
AP_FRONTEND_URL=${AP_FRONTEND_URL}

Reply:
yes this works, there's an `env_file` argument that points to the folder, you can test with a quick `docker compose up`